From de93f3af4b55549f33a1c4dcb67f6286da54b4d0 Mon Sep 17 00:00:00 2001
From: Liu Gang <Gang.Liu@nxp.com>
Date: Tue, 19 Apr 2016 10:37:53 +0800
Subject: [PATCH 041/245] layerscape/dts: Update GPIO node

commit 62a8d1cd36b53b69e1814f1f24821ae43cc88307
[doesn't apply arch/arm64/boot/dts/freescale/fsl-ls1043a-qds.dts]
[doesn't apply arch/arm64/boot/dts/freescale/fsl-ls2080a.dtsi]
[doesn't apply arch/arm64/boot/dts/freescale/fsl-ls2085a.dtsi]

For ls1043a and ls2085a, the GPIO compatible should be
changed to "fsl,qoriq-gpio".

For ls2080a and ls2085a, the little-endian property
should be added since the GPIO register working on
little endian mode on the two platforms.

Signed-off-by: Liu Gang <Gang.Liu@nxp.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
index 5c6fb76..60beb85 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -602,8 +602,8 @@
 			clocks = <&clockgen 4 0>;
 		};
 
-		gpio1: gpio@2300000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio0: gpio@2300000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2300000 0x0 0x10000>;
 			interrupts = <0 66 0x4>;
 			gpio-controller;
@@ -612,8 +612,8 @@
 			#interrupt-cells = <2>;
 		};
 
-		gpio2: gpio@2310000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio1: gpio@2310000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2310000 0x0 0x10000>;
 			interrupts = <0 67 0x4>;
 			gpio-controller;
@@ -622,8 +622,8 @@
 			#interrupt-cells = <2>;
 		};
 
-		gpio3: gpio@2320000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio2: gpio@2320000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2320000 0x0 0x10000>;
 			interrupts = <0 68 0x4>;
 			gpio-controller;
@@ -632,8 +632,8 @@
 			#interrupt-cells = <2>;
 		};
 
-		gpio4: gpio@2330000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio3: gpio@2330000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2330000 0x0 0x10000>;
 			interrupts = <0 134 0x4>;
 			gpio-controller;
-- 
2.1.0.27.g96db324

